### PR TITLE
add conscious

### PR DESCRIPTION
**Pull request by ithfay**

### CODE REVIEW

File: conscious/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "Conscious",
```
Comment (JeremyParish69) [inline conscious/assetlist.json:3]: The chain_name property cannot have capital letters

File: conscious/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "cvnt",
+          "exponent": 18
+        }
+      ],
+      "base": "acvn",
+      "name": "Cvn",
+      "display": "cvn",
```
Comment (JeremyParish69) [inline conscious/assetlist.json:19]: this display denom is not defined. Do you mean 'cvnt'?

File: conscious/chain.json [hunk 3]
```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Conscious",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.consciousdao.com/",
+  "pretty_name": "Cvn",
```
Comment (JeremyParish69) [inline conscious/chain.json:7]: Do you mean for this to display as 'ConsciousDAO'?

File: conscious/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        {
+          "denom": "acvnt",
+          "exponent": 0
+        },
+        {
+          "denom": "cvnt",
+          "exponent": 18
+        }
+      ],
+      "base": "acvn",
```
Comment (JeremyParish69) [inline conscious/assetlist.json:17]: this denom is not defined. Is it supposed to be acvnt?

File: conscious/chain.json [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+      {
+        "denom": "acvnt"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/cvn-network/cvn",
+    "recommended_version": "v2.0.0",
+    "compatible_versions": [
+      "v1.0.0"
```
Comment (JeremyParish69) [inline conscious/chain.json:35]: is this meant to me v2.0.0?

**Comment by JeremyParish69** [comment]
Please address the same concerns that were brought up in the old PR https://github.com/cosmos/chain-registry/pull/3253

**Comment by ithfay** [comment]
update conscious info

**Comment by ithfay** [comment]
update conscious info

**Comment by ithfay** [comment]
update conscious info

File: conscious/chain.json [hunk 6]
```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Conscious",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.consciousdao.com/",
+  "pretty_name": "ConsciousDAO",
+  "chain_id": "cvn_2032-1",
+  "bech32_prefix": "cvn",
```
Comment (JeremyParish69) [inline conscious/chain.json:9]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: conscious/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+          "denom": "cvnt",
+          "exponent": 18
+        }
+      ],
+      "base": "acvnt",
+      "name": "Cvn",
+      "display": "cvnt",
+      "symbol": "CVN",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/conscious/images/cvn.svg",
```
Comment (JeremyParish69) [inline conscious/assetlist.json:22]: Again, these images (which the png is square while the svg is not) seem unintentionally low quality/pixelated. Do we have clearer images?

File: conscious/chain.json (hunk 6 above)
Comment (ithfay) [reply conscious/chain.json:9]: I have registered this prefix.
https://github.com/satoshilabs/slips/pull/1641

Comment (JeremyParish69) [reply conscious/chain.json:9]: Marked as draft. Will proceed once merged.

Comment (ithfay) [reply conscious/chain.json:9]: OK.

**Comment by ithfay** [comment]
update info

**Comment by ithfay** [comment]
update info

File: conscious/chain.json [hunk 8]
```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Conscious",
```
Comment (JeremyParish69) [inline conscious/chain.json:3]: please correct chain_name to 'conscious' (lowercase 'c')

Comment (JeremyParish69) [reply conscious/chain.json]: Updated this myself

**Pull request merged**